README.rst

```diff
@@ -547,6 +532,112 @@ network topology module will trigger the monitoring checks.
 For more information see:
 `Network Topology Device Integration <https://github.com/openwisp/openwisp-network-topology#integration-with-openwisp-controller-and-openwisp-monitoring>`_
 
+``OPENWISP_MONITORING_MAC_VENDOR_DETECTION``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``True``    |
++--------------+-------------+
+
+Indicates whether mac addresses will be complemented with hardware vendor
+information by performing lookups on the OUI
+(Organization Unique Identifier) table.
+
+This feature is enabled by default.
+
+``OPENWISP_MONITORING_METRICS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``dict``    |
++--------------+-------------+
+| **default**: | ``{}``      |
++--------------+-------------+
+
+This setting allows to define additional metric configuration or to override
+the default metric configuration defined in
+``openwisp_monitoring.monitoring.configuration.DEFAULT_METRICS``.
+
+For example, if you want to change only the **field_name** of
+``clients`` metric to ``wifi_clients`` you can use:
+
+.. code-block:: python
+
+    from django.utils.translation import gettext_lazy as _
+
+    OPENWISP_MONITORING_METRICS = {
+        'clients': {
+            'label': _('WiFi clients'),
+            'field_name': 'wifi_clients',
+        },
+    }
+
+For example, if you want to change only the default alert settings of
+``memory`` metric you can use:
+
+.. code-block:: python
+
+    OPENWISP_MONITORING_METRICS = {
+        'memory': {
+            'alert_settings': {'threshold': 0.75, 'tolerance': 5}
+        },
+    }
+
+For example, if you want to change only the notification of
+``config_applied`` metric you can use:
+
+.. code-block:: python
+
+    from django.utils.translation import gettext_lazy as _
+
+    OPENWISP_MONITORING_METRICS = {
+        'config_applied': {
+            'notification': {
+                'problem': {
+                    'verb': _('has not been applied'),
```

Comment:
does it have any effect changing `verbose_name` here? If it does, please include it in the example, eg: "PING problem changed name" or something like that.

Reply:
Yes, it will have. Earlier I wasn't very sure how helpful it might be earlier. It should be after `Notification Settings`, I think. Added the `verbose_name` as `Configuration PROBLEM` (default is `Configuration Applied PROBLEM` so just shortened it).